### DISCUSSION TITLE
What to back-up before enabling an experimental setting?

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [FreeTubeApp/FreeTube#3026] [Q&A]
![image](https://user-images.githubusercontent.com/47056543/210297116-184a4759-68c4-4028-aad1-6b517af40a71.png)

What to backup, though? On Linux, is it `~/.config/FreeTube`? And on Windows .  . ?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @LinuxOnTheDesktop i think info u need is on https://docs.freetubeapp.io/usage/data-location/

Feel free to ask if u need more info :)
